config/projects/chefdk.rb

```diff
@@ -32,8 +32,8 @@
 end
 
 # Uncomment to pin the chef version
-override :chef,           version: "stable"
-override :ohai,           version: 'stable'
+override :chef,           version: "12.4-stable"
```

Comment:
I'd like to build both chef and chef-dk, yeah--lets us get coverage of this new build on delivery.